### ISSUE TITLE
External primary monitor behaves as secondary only after boot

### ISSUE THREAD

**Issue by duarch** [pop-os/pop#1480]
**Distribution (run `cat /etc/os-release`):** 
``` 
NAME="Pop!_OS"
VERSION="20.10"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.10"
VERSION_ID="20.10"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=groovy
UBUNTU_CODENAME=groovy
LOGO=distributor-logo-pop-os
``` 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
Not the case

**Issue/Bug Description:**
Even if I choose my external monitor as Primary Display it behaves as second or unavailable to show the login prompt UI. 

![POP_afterboot](https://user-images.githubusercontent.com/42032253/104130285-96f31680-534e-11eb-8e75-bfaecdf7cef8.jpeg)
PS. it is possible to see the mouse pointer.

**Steps to reproduce (if you know):**
Just boot

**Expected behavior:** As other distros I expected the login move along with the mouse movement or at least appear on the main selected monitor. 
**deepin**
![deepin](https://user-images.githubusercontent.com/42032253/104129800-95285380-534c-11eb-94a1-2f8613d0d7fe.jpeg)

**elementary OS**
![elementaryOS](https://user-images.githubusercontent.com/42032253/104130219-4c719a00-534e-11eb-9881-4b158e736c4f.jpeg)

**UbuntuDDE**
![UbuntuDDE](https://user-images.githubusercontent.com/42032253/104130258-71fea380-534e-11eb-985e-df67d344737e.jpeg)

As it behaves after locking the screen.
**Pop OS Lock Screen**
![POP_lockScreen](https://user-images.githubusercontent.com/42032253/104130278-8a6ebe00-534e-11eb-9ef9-d7ccc95cee03.jpeg)

**Other Notes:**
Believing that such behavior was related to  **HiDPI Daemon** I tried multiple combinations, but all of them keeps the same.

Not related with NVIDIA card (I don't have it)

**Comment by Zwiebelsaft** [comment]
I have encountered the same issue. I think you can, in the terminal, make it so it prioritizes, setting up the monitor correctly and it uses the right config earlier. I don't know how that works though. I just seen one youtube video long time ago about that and I remember it being pretty complicated. I wonder if the makers of Pop OS actually read this. I hope they do.
https://www.youtube.com/watch?v=lhiLWxJgiAo
maybe this helps

**Comment by bflanagin** [comment]
@duarch for clarification, it is setting the external display as primary once you have logged in?

**Comment by ArjunSingh99** [comment]
I have this exact same issue too. I think this is related to the fact that your configuration isn't loaded till you're logged in. For example, themes aren't loaded either, the monitor position is set to the right side for me, even though it's set to left side for me. And even mouse acceleration is reverted to default in this state.

**Comment by duarch** [comment]
> @duarch for clarification, it is setting the external display as primary once you have logged in?

@bflanagin Yes 

> **Issue/Bug Description:**
> Even if I choose my external monitor as Primary Display it behaves as second or unavailable to show the login prompt UI.

**Comment by bflanagin** [comment]
@duarch I've asked around, and I've got good news and bad news. 

Good news is there is a work around:
https://askubuntu.com/questions/1255230/ubuntu-20-04-login-screen-doesnt-appear-on-the-primary-screen (will work with 20.10 as well)

Bad news is that it seems to be a bug with the version of GDM that is shipping in Pop!_OS at the moment. Sense we try to stay in sync with upstream as much as possible (without breaking user experience, introducing major bugs, etc.) we'll have to wait till upstream GDM fixes the issue. 

Closing ticket for now.
